.github/workflows/manual-deploy-obscuro-testnet-network.yml

```diff
@@ -208,4 +211,35 @@ jobs:
               --ip-config-name ipconfigObscuroNodeTestnet-0-${{ GITHUB.RUN_NUMBER }} \
               --nic-name ObscuroNodeTestnet-0-${{ GITHUB.RUN_NUMBER }}VMNic \
               --resource-group Testnet \
-              --lb-name testnet-loadbalancer
\ No newline at end of file
+              --lb-name testnet-loadbalancer
+
+  setup:
+    needs: deploy
```

Comment:
Because this depends on `deploy` it should run simultaneously with the `update` step above. This is fine, we can deploy contracts on an L2 node while the loadbalancer is setup and the old nodes are deleted. (But we have to be careful that this step never requires the loadbalancer, it should talk to nodes directly).